### ISSUE TITLE
Failed to use the standard mergetool

### ISSUE THREAD

**Issue by etomm** [configit-open-source/csmerge#23]
Would you like to resolve the conflict with the mergetool?
y
An error occurred:
System.NullReferenceException: Object reference not set to an instance of an object.
   at Integration.GitHelper.GetMergeCmdLine(Repository repository) in C:\projects\csmerge\NuGetHelpers\GitHelper.cs:line 19
   at Integration.GitHelper.RunStandardMergetool(Repository repository, String base, String local, String incoming, String resolved) in C:\projects\csmerge\NuGetHelpers\GitHelper.cs:line 25
   at Integration.GitHelper.ResolveWithStandardMergetool(Repository repository, String fullConflictPath, XDocument baseContent, XDocument localContent, XDocument incomingContent, Logger logger, String conflict) in C:\projects\csmerge\NuGetHelpers\GitHelper.cs:line 133
   at CsMerge.CsMerge.ProcessProjectFiles(CurrentOperation operation, String[] conflictPaths, DirectoryInfo folder, Logger logger, String rootFolder) in C:\projects\csmerge\CsMerge\CsMerge.cs:line 365
   at CsMerge.CsMerge.ProcessMerge(Logger logger, DirectoryInfo folder, String rootFolder) in C:\projects\csmerge\CsMerge\CsMerge.cs:line 103
   at CsMerge.CsMerge.Main(String[] args) in C:\projects\csmerge\CsMerge\CsMerge.cs:line 55

**Comment by Scott-Harris** [comment]
This error occurs if you have not configured a mergetool cmd in your git config file.

See https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

As a work around, set the mergetool cmd in your git config.

For example (from link above):
```
[mergetool "extMerge"]
  cmd = extMerge "$BASE" "$LOCAL" "$REMOTE" "$MERGED"
```

**Comment by etomm** [comment]
I had that configured using perforce merge. And I am no more using csmerge because of it.

If you would like a test nowaday I will need to reinstall all and try. Do you need it?

**Comment by Scott-Harris** [comment]
This should be fixed in release [0.2.9](https://github.com/configit-open-source/csmerge/releases/tag/release%2F0.2.9).
